gemmbench/gemm_utils.py

```diff
@@ -180,9 +192,9 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
     with tk.gen.TestLaunchContext(
         hyperparams, canonicalize=True, run=True, run_config=config
     ):
-        a = torch.randn(shape[0], shape[2], dtype=dtype)
-        b = torch.randn(shape[1], shape[2], dtype=dtype)
-        c = torch.zeros(shape[0], shape[1], dtype=torch.float32)
+        a = torch.randn(shape[0], shape[2], operand_element_type=operand_element_type)
+        b = torch.randn(shape[1], shape[2], operand_element_type=operand_element_type)
+        c = torch.zeros(shape[0], shape[1], operand_element_type=torch.float32)
```

Comment:
Should we change from hardcoded torch.float32 to the accumulator type based on the operand type for tk?

Reply:
I think this should be fixed when tk supports more than the default. Maybe I'll add an assertion so that we don't forget about this.